src/components/game-room/boardMap/bubble/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { bubbleRightMap } from '@/constants/boardMap/bubbleRightMap';
+import { useGameDiceStore } from '@/stores/websocket/useGameDiceStore';
+
+import { UserCharacter } from '../GameBoard';
+import BubbleContent from './BubbleContent';
+
+interface BubbleProps {
+  isActiveDice: boolean;
+  character: UserCharacter;
+  charWidth: number;
+  characterX: number;
+  characterY: number;
+}
+
+const Bubble = ({
+  isActiveDice,
+  character,
+  charWidth,
+  characterX,
+  characterY,
+}: BubbleProps) => {
+  const { diceUsername } = useGameDiceStore();
+
+  const isLeftSide = bubbleRightMap[character.position] || false;
```

Comment:
bubbleRigthMap이라는 객체를 선언하는 것 보단, 
```suggestion
  const isLeftSide = character.position < 18
```

이라고 했어도 되지 않을까 싶습니다.